cmd/soroban-rpc/internal/preflight/preflight.go

```diff
@@ -224,6 +225,7 @@ func getInvokeHostFunctionPreflight(params PreflightParameters) (Preflight, erro
 		invokeHostFunctionCXDR,
 		sourceAccountCXDR,
 		li,
+		C.bool(params.EnableDebug),
```

Comment:
huh.. cgo supports c++ bool datatype ?!

I'm pleasantly surprised to see that.